dev.galasa.plugin.common/build.gradle

```diff
@@ -0,0 +1,169 @@
+                    licenses {
+                        license {
+                            name = 'Eclipse Public License - v 2.0'
+                            url = 'https://www.eclipse.org/legal/epl-2.0'
+                        }
+                    }
+                    developers {
+                        developer {
+                            name = 'Michael Baylis'
+                            email = 'Michael.Baylis@uk.ibm.com'
+                            organization = 'IBM'
+                            organizationUrl = 'https://www.ibm.com'
+                        }
+                        developer {
+                            name = 'William Yates'
+                            email = 'wyates@uk.ibm.com'
+                            organization = 'IBM'
+                            organizationUrl = 'https://www.ibm.com'
+                        }
+                        developer {
+                            name = 'Michael Cobbett'
+                            email = 'mike_cobbett@uk.ibm.com'
+                            organization = 'IBM'
+                            organizationUrl = 'https://www.ibm.com'
+                        }
+                        developer {
+                            name = 'Jade Carino'
+                            email = 'jade.carino@ibm.com'
+                            organization = 'IBM'
+                            organizationUrl = 'https://www.ibm.com'
+                        }
+                        developer {
+                            name = 'Savvas Kyriacou'
+                            email = 'savvasky@uk.ibm.com'
+                            organization = 'IBM'
+                            organizationUrl = 'https://www.ibm.com'
+                        }
+                        developer {
+                            name = 'Aaron Jhaj'
+                            email = 'aaron.jhaj@ibm.com'
+                            organization = 'IBM'
+                            organizationUrl = 'https://www.ibm.com'
+                        }
+                        developer {
+                            name = 'Eamonn Mansour'
+                            email = 'eamonn.mansour@ibm.com'
+                            organization = 'IBM'
+                            organizationUrl = 'https://www.ibm.com'
+                        }
+                    }
+                    scm {
+                        connection = 'scm:git:git:://github.com/galasa-dev/gradle'
+                        developerConnection = 'scm:git:git:://github.com/galasa-dev/gradle'
+                        url = 'https://github.com/galasa-dev/gradle'
+                    }
+                    issueManagement {
+                        system = 'GitHub'
+                        url = 'https://github.com/galasa-dev/projectmanagement/issues'
+                    }
+                }
+            }
+        }
+    }
+    repositories {
+        maven {
+            url  = "$targetMaven"
```

Comment:
Maybe I'm not understanding what this does with Maven Central, can we discuss?

Reply:
Discussed on a call. Fixed by removing gradle properties which default the targetMaven value erroneously.